clap_mangen/src/render.rs

```diff
@@ -61,7 +61,11 @@ pub(crate) fn synopsis(roff: &mut Roff, cmd: &clap::Command) {
     for arg in cmd.get_positionals() {
         let (lhs, rhs) = option_markers(arg);
         line.push(roman(lhs));
-        line.push(italic(arg.get_id()));
+        if let Some(value) = arg.get_value_names() {
+            line.push(bold(value.join(" ")));
+        } else {
+            line.push(bold(arg.get_id()));
+        }
```

Comment:
How come this was changed from italic to bold?

Reply:
I was trying to mimic the man page for `git-diff`. However, I just found out that italic denotes replaceable arguments by convention. From https://linux.die.net/man/7/man-pages

>SYNOPSIS
>
>briefly describes the command or function's interface. For commands, this shows the syntax of the command and its arguments (including options); boldface is used for as-is text and italics are used to indicate replaceable arguments. Brackets ([]) surround optional arguments, vertical bars (|) separate choices, and ellipses (...) can be repeated. For functions, it shows any required data declarations or #include directives, followed by the function declaration.

As I am reading this, I can also see that angle brackets i.e `<>` should not be used in the synopsis section. I will try to address this in a subsequent PR.